salvo/src/lib/api/docker_volume.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package salvo;
+
+// This message defines the properties for a given mount. It is used
+// to generate the dictionary specifying volumes for the Python Docker
+// SDK
+message VolumeProperties {
+  // Defines a list of properties governing the mount in the container
+  string bind = 1;
+
+  // Define whether the mount point is read-write or read-only
+  string mode = 2;
+}
+
+// This message defines the volume structure consumed by the command
+// to run a docker image.
+message Volume {
+  // Specify a map of volumes and their mount points for use in a container
+  map<string, VolumeProperties> volumes = 1;
```

Comment:
The field name suggests that the volume contains a set of volumes? Is that the architecture?

Or is it that the volume contains a repeated set of volume properties? In which case we may want to call the field `volume_properties`.